app/controllers/chargeback_controller.rb

```diff
@@ -134,14 +134,12 @@ def cb_rate_edit
       # Detect errors saving tiers
       tiers_valid = @rate_tiers.all? { |tiers| tiers.all?(&:valid?) }
 
-      @rate.chargeback_rate_details.replace(@rate_details)
```

Comment:
@lpichler, is a merge error. In the commit Fix rubocop warnings I pushed a code that you was editing. Sorry. How I can proceed to fix this?